src/scenes/BagScene.ts

```diff
@@ -0,0 +1,160 @@
+import Phaser from "phaser"
+
+import { Events, sceneEvents } from "../events/EventsCenter"
+import SettingsMenu from "./SettingsMenu"
+import { Dialog } from "@/ui/Dialog"
+import { Tip } from "@/ui/Tip"
+import { SoundSingleton, SoundEffects } from "@/utils/SoundSingleton"
+import { GameOverModal } from "@/ui/GameOverModal"
+import { WinMarioLikeLevelModal } from "@/ui/WinMarioLikeLevelModal"
+import { Modal, ReceivesInstructions } from "@/types/Modal"
+
+export default class BagScene extends Phaser.Scene {
```

Comment:
Looks like this Scene has the same stuff present in GameUI Scene (hearts, berries, etc) => This will be used or cane be removed? (Probably you copied the code)